src/current/v23.2/molt-fetch.md

```diff
@@ -0,0 +1,413 @@
+
+{% include_cached copy-clipboard.html %}
+~~~
+--local-path /migration/data/cockroach
+--local-path-listen-addr 'localhost:3000'
+--local-path-crdb-access-addr '44.55.66.77:3000'
+~~~
+
+A local file server can be used with either the [`IMPORT INTO` or `COPY FROM` modes](#fetch-mode).
+
+{{site.data.alerts.callout_success}}
+[Cloud storage](#cloud-storage) is often preferable to a local file server, which can require considerable disk space.
+{{site.data.alerts.end}}
+
+#### Direct copy
+
+`--direct-copy` specifies that MOLT Fetch should use `COPY FROM` to move the source data directly to CockroachDB without an intermediate store:
+
+- Because the data is held in memory, the machine must have sufficient RAM for the amount of data being moved.
+- Direct copy mode does not support compression or [continuation](#fetch-continuation).
+- The [`--use-copy`](#fetch-mode) flag is redundant with `--direct-copy`.
+
+### Schema and table selection
+
+By default, MOLT Fetch moves all source data to CockroachDB. Use the following flags to move a subset of data.
+
+`--schema-filter` specifies a range of schema objects to move to CockroachDB, based on a regex string. For example, to move every table in the source database's `public` schema:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--schema-filter 'public'
+~~~
+
+`--table-filter` specifies a range of tables to move to CockroachDB, based on a regex string. For example, to move every table in the source database that has "user" in the title:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--table-filter '\w*user\w*'
+~~~
+
+### Target table handling
+
+`--table-handling` defines how MOLT Fetch loads data on the CockroachDB tables that [match the selection](#schema-and-table-selection).
+
+To load the data without changing the existing data in the tables, use `'none'`:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--table-handling 'none'
+~~~
+
+To [truncate]({% link {{ page.version.version }}/truncate.md %}) tables before loading the data, use `'truncate-if-exists'`:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--table-handling 'truncate-if-exists'
+~~~
+
+To drop existing tables and create new tables before loading the data, use `'drop-on-target-and-recreate'`:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--table-handling 'drop-on-target-and-recreate'
+~~~
+
+With each option, MOLT Fetch creates a new CcokroachDB table to load the source data if one does not exist.
+
+### Fetch continuation
+
+If `molt fetch` exits with an error after being run, you can continue the process from the *continuation point* where it was interrupted.
+
+To retry all data starting from the continuation point, include `--fetch-id` and specify the process ID from the `molt fetch` output.
```

Comment:
Maybe wroth clarifying here that this is during the import phase of the data. If an error occurs during the export time, it starts from the beginning.

Reply:
What is the difference between the import and export phases? I wasn't aware of that distinction.